app/components/submission.js

```diff
@@ -0,0 +1,172 @@
+// https://github.com/EnCiv/undebate-ssp/issues/51
+
+import React from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+import SvgDeadlineMissed from '../svgr/deadline-missed'
+import SvgCheck from '../svgr/check'
+import SvgReminder from '../svgr/reminder-sent'
+import SvgFeelingBlue from '../svgr/feeling-blue'
+
+export default function Submission(props) {
+    const { className, style, electionObj } = props
+    const classes = useStyles(props)
+
+    const getSubmission = () => {
+        const sortedSubmissions = electionObj.moderator.submissions.sort((a, b) => b.date - a.date)
```

Comment:
dates are strings, JS does not subtract strings well.  Use this 
```
list.sort(function (a, b) {
    return ('' + a.date).localeCompare(b.date);
})
```
But check the order you end up with. It might need to be reversed. 

ref: https://stackoverflow.com/questions/51165/how-to-sort-strings-in-javascript